index.html

```diff
@@ -264,41 +261,47 @@ <h3>
         (e.g. in a native wake lock framework) or by the user agent, if it has
         direct hardware control.
       </p>
-      <p>
-        Each <a>platform wake lock</a> (one per <a>wake lock type</a>) has an
-        associated <dfn>state record</dfn> per {{Document}} with the following
-        <a data-cite="ECMASCRIPT#sec-object-internal-methods-and-internal-slots">internal
-        slots</a>:
-      </p>
-      <table class="simple">
-        <thead>
-          <tr>
-            <th>
-              Internal slot
-            </th>
-            <th>
-              Initial value
-            </th>
-            <th>
-              Description (<em>non-normative</em>)
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td>
-              <dfn>[[\ActiveLocks]]</dfn>
-            </td>
-            <td>
-              The empty <a>list</a>.
-            </td>
-            <td>
-              A <a>list</a> of {{WakeLockSentinel}} objects, representing
-              active wake locks associated with a {{Document}}.
-            </td>
-          </tr>
-        </tbody>
-      </table>
+    </section>
+    <section data-dfn-for="Document">
+      <h2>
+        Extensions to the `Document` interface
+      </h2>
+      <section>
+        <h3>
+          Internal slots
+        </h3>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>
+                Internal slot
+              </th>
+              <th>
+                Initial value
+              </th>
+              <th>
+                Description (<em>non-normative</em>)
```

Comment:
The description gives us the normative type, so... 
```suggestion
                Description
```

Reply:
Oops, read the columns out of order... but I think it might still be ok to say the description is normative here.

Reply:
Done, thanks.